packages/editor/src/store/actions.js

```diff
@@ -57,6 +73,83 @@ export function resetAutosave( post ) {
 	};
 }
 
+/**
+ * Optimistic action for dispatching that a post update request has started.
+ *
+ * @param {Object} options
+ * @return {Object} An action object
+ */
+export function __experimentalRequestPostUpdateStart( options = {} ) {
+	return {
+		type: 'REQUEST_POST_UPDATE_START',
+		optimist: { type: BEGIN, id: POST_UPDATE_TRANSACTION_ID },
+		options,
+	};
+}
+
+/**
+ * Optimistic action for indicating that the request post update has completed
+ * successfully.
+ *
+ * @param {Object} previousPost The previous post prior to update.
```

Comment:
There's only one parameter to this function: an object. If the properties of the object are to be documented, they should be as prescribed by JSDoc and in the JavaScript Documentation Standards, e.g. `options.previousPost`

http://usejsdoc.org/tags-param.html#parameters-with-properties
https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

There are multiple instances elsewhere in the pull request.

Reply:
> There's only one parameter to this function: an object.

Sorry I don't follow.  There's multiple arguments on this function not one.

Reply:
Oh I follow now.  I think this was the doc block from the original function that I just copied over (but I could be wrong), I'll fix.